clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/EhcacheMessageType.java

```diff
@@ -35,6 +35,7 @@
   VALIDATE_SERVER_STORE,
   RELEASE_SERVER_STORE,
   DESTROY_SERVER_STORE,
```

Comment:
`CREATE_SERVER_STORE` & `DESTROY_SERVER_STORE` are not relevant message types anymore, right ?